src/mock-data/posts.json

```diff
@@ -1,4 +1,3076 @@
 [
+  {
+    "id": "62502a05e538880600e2f1a3",
+    "style": "article:v2:default",
+    "slug": "russian-invasion-of-ukraine-2022-kid-frequently-asked-questions",
+    "hero_image": {
+      "filetype": "image/jpeg",
+      "resized_targets": {
+        "mobile": {
+          "height": 450,
+          "width": 800,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-mobile.jpg"
+        },
+        "tiny": {
+          "height": 84,
+          "width": 150,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-tiny.jpg"
+        },
+        "desktop": {
+          "height": 900,
+          "width": 1600,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-desktop.jpg"
+        },
+        "tablet": {
+          "height": 675,
+          "width": 1200,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-tablet.jpg"
+        },
+        "w400": {
+          "height": 225,
+          "width": 400,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-w400.jpg"
+        }
+      },
+      "id": "625563aa96c54f0600c96842",
+      "description": "俄烏戰爭兒童報舉手發問主圖"
+    },
+    "og_image": {
+      "filetype": "image/jpeg",
+      "resized_targets": {
+        "mobile": {
+          "height": 450,
+          "width": 800,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-mobile.jpg"
+        },
+        "tiny": {
+          "height": 84,
+          "width": 150,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-tiny.jpg"
+        },
+        "desktop": {
+          "height": 900,
+          "width": 1600,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-desktop.jpg"
+        },
+        "tablet": {
+          "height": 675,
+          "width": 1200,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-tablet.jpg"
+        },
+        "w400": {
+          "height": 225,
+          "width": 400,
+          "url": "https://www.twreporter.org/images/20220412193402-8749a8649a48cc4977853b930d802bf7-w400.jpg"
+        }
+      },
+      "id": "625563aa96c54f0600c96842",
+      "description": "俄烏戰爭兒童報舉手發問主圖"
+    },
+    "og_description": "《報導者》與11所國中、小學合作，收到220名、2到9年級學生的提問，整理出20個台灣孩子對於俄羅斯入侵烏克蘭戰爭最想知道的事。我們請專家解釋，也有來自記者國際採訪的第一手資訊，為大家整理出答案。",
+    "title": "台灣孩子大提問：關於俄羅斯侵略烏克蘭戰爭，大家最想知道的是⋯⋯",
+    "subtitle": "你問我答",
+    "published_date": "2022-04-13T16:00:00Z",
+    "is_external": false,
+    "tags": [
```

Comment:
Could you help to add some category/subcategory info in `category_set` field in order to show this article in category page if needed ?

Reply:
Sure, but I'm not sure where is the correct `category/subcategory` in the post, could you give me some hint about it ?
Thanks @Aylie-Chou.

Reply:
category/subcategory id could be fould in [core package](https://github.com/twreporter/twreporter-npm-packages/blob/master/packages/core/src/constants/category-set.js).

Reply:
Updated ([11702e6](https://github.com/twreporter/twreporter-react/pull/2227/commits/11702e67be1e8bccd56836da308d8dc2d6464720)).

Since the mock post (slug:[ russian-invasion-of-ukraine-2022-kid-frequently-asked-questions](https://www.twreporter.org/a/russian-invasion-of-ukraine-2022-kid-frequently-asked-questions)) does not belong to any categories, I'm not pretty sure which one is its `category_set`.

Nevertheless, I suppose the `category_set` of this post would be 
```
        "category": {
          "id": "63206383207bf7c5f8716260",
          "name": "教育校園"
        },
        "subcategory": {
          "id": "63206383207bf7c5f8716263",
          "name": "青少年"
        }
```

Please let me know if it need to be changed, thanks 😉